library/xml-model/src/test/java/org/cru/godtools/xml/model/tract/TractPageTest.kt

```diff
@@ -59,5 +61,15 @@ class TractPageTest {
         assertEquals(Color.BLUE, TractPage(Manifest(cardBackgroundColor = Color.BLUE)).cardBackgroundColor)
     }
 
+    @Test
+    fun testTextScale() {
```

Comment:
Should this test use the string format for the test name(ex. ```fun `parseManifest - Lesson Manifest`()```), or are we only using that on certain tests?

Reply:
I haven't established a best practice with it yet, I've been most experimenting with the space separate name style

Reply:
also, ```fun `testTextScale`()``` will cause a line error because the ``` ` ``` character isn't necessary for the name